README.md

```diff
@@ -78,8 +78,11 @@ On CoreOS Container Linux, the supported providers and metadata are [somewhat di
     - Attributes
       - AFTERBURN_PACKET_HOSTNAME
       - AFTERBURN_PACKET_IPV4_PUBLIC_0
+      - AFTERBURN_PACKET_IPV4_PUBLIC_GW_0
       - AFTERBURN_PACKET_IPV4_PRIVATE_0
+      - AFTERBURN_PACKET_IPV4_PRIVATE_GW_0
       - AFTERBURN_PACKET_IPV6_PUBLIC_0
+      - AFTERBURN_PACKET_IPV6_PUBLIC_GW_0
```

Comment:
`…IPV6_PRIVATE…` is missing, or?

Reply:
Nope, there is no private IPv6 address on Packet. I've included a test for it because there is a code path for it, which is supposed to work should Packet ever add private IPv6 addresses.